lib/src/golden_test_runner.dart

```diff
@@ -30,6 +30,7 @@ abstract class GoldenTestRunner {
     required Widget widget,
     required ThemeData? globalConfigTheme,
     required ThemeData? variantConfigTheme,
+    required GoldenTestTheme? goldenTestTheme,
```

Comment:
Interesting that I didn't get a lint warning me that I had added a parameter to an overridden method that wasn't present in the paren't method 🤔